pallets/symmetric-key/Cargo.toml

```diff
@@ -24,6 +24,7 @@ sp-std = { default-features = false, version = "4.0.0", git = "https://github.co
 serde = { version = "1.0.137" }
 frame-support-test = { version = "3.0.0", git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.32" }
 pallet-scheduler = { default-features = false, version = "4.0.0-dev", git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.32" }
+pallet-preimage = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.32" }
```

Comment:
pallet-preimage isn't necessary here (and actually isn't being used)

Reply:
removed